index.html

```diff
@@ -1,14 +1,60 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="kor">
```

Comment:
```suggestion
<html lang="ko">
```
사소한 부분이지만... 이렇게 쓰는게 올바른 방법입니다.

Reply:
감사합니다. 반영하겠습니다!!